lib/babe/verify_test.go

```diff
@@ -84,23 +92,79 @@ func TestVerifySlotWinner(t *testing.T) {
 }
 
 func TestVerifyAuthorshipRight(t *testing.T) {
-	babesession := createTestSession(t, nil)
-	err := babesession.configurationFromRuntime()
-	if err != nil {
-		t.Fatal(err)
+	testsCases := []struct {
+		description                     string
+		parentHeader                    *types.Header
+		expectedErr                     error
+		authorshipRight                 bool
+		expectedErrAfterAuthorshipRight error
+	}{
+		{
+			description:                     "test verify block with existing parent",
```

Comment:
I'm not sure what this description means? we don't need to test the `AddBlock` function here

Reply:
oh, well when I started this PR, the test was using a block parent that was not persisted into the database. 
There are then two tests now, the old one: `test verify block with not existing parent` that should fail with `cannot find parent block in blocktree` and then the new one for the new testcase: `test verify block with existing parent` that does not fail with that error on that step, but should fail further on with `block producer equivocated`.

Reply:
okay, I don't think it's really necessary to test but it's ok to leave it in